lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -141,6 +148,11 @@ sub register {
           ->to('Plugin::ObsRsync::Controller::Folders#list');
         $plugin_api_r->put('/obs_rsync/#project/runs')->name('plugin_obs_rsync_api_run')
           ->to('Plugin::ObsRsync::Controller::Gru#run');
+
+        $plugin_api_r->get('/obs_rsync/#alias/latest_test')->name('plugin_obs_rsync_api_latest_test')
+          ->to('Plugin::ObsRsync::Controller::Folders#latest_test');
+        $plugin_api_r->get('/obs_rsync/#alias/latest_test?full=#full')->name('plugin_obs_rsync_api_latest_test_full')
+          ->to('Plugin::ObsRsync::Controller::Folders#latest_test');
```

Comment:
This seems to be a verbatim copy of the above. Is this intended?

Reply:
Yes, to let the same routes be available for humans and robots (OpenID vs API authentication).

Reply:
ok, then how about you extract a helper method that adds these routes and you call that instead? can make the intention clearer and prevent running out of sync